images/base/files/usr/local/bin/entrypoint

```diff
@@ -179,6 +179,15 @@ enable_network_magic(){
   # now we can ensure that DNS is configured to use our IP
   cp /etc/resolv.conf /etc/resolv.conf.original
   sed -e "s/${docker_embedded_dns_ip}/${docker_host_ip}/g" /etc/resolv.conf.original >/etc/resolv.conf
+
+  # fixup IPs in manifests ...
+  # TODO: dual-stack LOL :this-is-fine:
+  curr_ip=$(hostname --ip-address)
```

Comment:
we'll want to swap this out with something else. this was sufficient for bootstrapping it but we'll need to actually change this to look at /kind/old-ip and get an IP of the same family.

not going to be too hard, but wasn't relevant to the bigger problems. bash wrangling for ipv6 singlestack can be a follow up, but should happen prior to releasing any official kind updates.